src/rules/at-rule-blockless-newline-after/__tests__/index.js

```diff
@@ -0,0 +1,27 @@
+import {
+  ruleTester,
+} from "../../../testUtils"
+import rule, { ruleName, messages } from ".."
+
+const testRule = ruleTester(rule, ruleName)
+
+testRule("always", tr => {
+  tr.ok("@import url(\"x.css\");\n")
```

Comment:
Can each of these tests be followed by either a rule or another blockless `@rule` please.